mautic/mautic
# Provision to sort editor fonts

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ❌ 
| New feature/enhancement? (use the a.x branch)      | ✅ 
| Deprecations?                          | ❌ 
| BC breaks? (use the c.x branch)        | ❌ 
| Automated tests included?              | ❌  
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



This PR introduces provision to use sorted editor fonts in JS scripts. Also, render unescaped security authentication content.

#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Change order of editor_fonts in local.php
3. Observe sorted fonts in page source




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11389"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
